app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new
+    @like.author_id = @user.id
+    @like.post_id = @post.id
```

Comment:
- [ ] Hi you did a nice job but there is an issue when trying to like a post. This is because the variable `@user` has not been declared so it doesn't have an `id`. I'll recommend using the current user id to create the new like i.e `@like.author_id = current_user.id`